test/TestUtil.js

```diff
@@ -23,15 +23,18 @@ TestUtil.testBooleanFunction = function (simplifier, exprString, expectedBoolean
 };
 
 // Tests a simplification function
-TestUtil.testSimplification = function (simplifyingFunction, exprString, expectedOutputString) {
+TestUtil.testSimplification = function (simplifyingFunction, exprString,
+                                        expectedOutputString) {
   it (exprString + ' -> ' + expectedOutputString,  () => {
     assert.deepEqual(
-      print(simplifyingFunction(flatten(math.parse(exprString))).newNode),expectedOutputString);
+      print(simplifyingFunction(
```

Comment:
make this line the first thing you're checking for equality. ie `simplifyingFunction( flatten(math.parse(exprString))).newNode)` (if that fits), and then the next line `expectedOutputString`